src/App.js

```diff
@@ -1,5 +1,65 @@
+import { Console } from "@woowacourse/mission-utils";
+
 class App {
-  async run() {}
+  async run() {
+    try {
+      let input = await Console.readLineAsync("덧셈할 문자열을 입력해주세요.");
+
```

Comment:
```suggestion
      let input = await Console.readLineAsync("덧셈할 문자열을 입력해주세요.\n");

```
문제에서 입출력 예시보면 한 칸 들여쓰기 후 입력을 받기 때문에 '\n'을 추가하면 더 좋을 것 같습니다.